stdlib/universe/filter.go

```diff
@@ -194,14 +194,20 @@ func (t *filterTransformation) Process(id execute.DatasetID, tbl flux.Table) err
 	}
 
 	// Filter the table and pass in the indices we have to read.
-	table, err := t.filterTable(tbl, record, indices)
+	table, done, err := t.filterTable(tbl, record, indices)
 	if err != nil {
 		return err
 	} else if table.Empty() && !t.keepEmptyTables {
 		// Drop the table.
 		return nil
 	}
-	return t.d.Process(table)
+	if err := t.d.Process(table); err != nil {
+		return err
+	}
+	// Wait until the full table has been processed and read by the next transformation.
+	// Filter can only process one table at a time.
+	<-done
```

Comment:
I've the same question as above here.  Don't we want to see if the context was canceled?